modules/combined/doc/content/newsletter/2018_08_17.md

```diff
@@ -0,0 +1,95 @@
+                    {'x':[1,2,3,4], 'y':[1,4,9,16], 'name':'x*x', 'marker':{'size':14}}]
+
+## MeshModifier for creating lower dimensional blocks
+
+It is now possible to generate a lower dimensional block from sidesets using the
+MOOSE MeshModifier `LowerDBlockFromSideset`. The required parameters to use the
+modifier are `sidesets` which is a vector of `BoundaryID`s from which to build
+the new block and `new_block_id` which is the `SubdomainID` of the new block. An
+optional parameter is `new_block_name` if the user wants to define a name for
+the newly created block. The order of the lower dimensional elements created
+will match the order of the parent element. E.g. if we are creating a block from
+`QUAD9` or `TRI6` faces, then the newly created 1D block will have elements of type
+`EDGE3`. `QUAD4` and `TRI3` elements would yield `EDGE2`. 3D behavior is
+analogous. The snippet below shows input file syntax for using
+`LowerDBlocksFromSideset`:
+
+```
+[MeshModifiers]
+  [./lower]
+    type = LowerDBlockFromSideset
+    sidesets = '0 1 2 3'
+    new_block_id = 10
+    new_block_name = 'lower_d'
+  [../]
+[]
+```
+
+
+## Threaded General User Objects
+
+Up until now, general user objects were executed in a single threaded region, even if the
+application ran with multiple-threads.  Now, it is possible to have threaded general user objects.
+Users need to inherit from `ThreadedGeneralUserObject` class and MOOSE will create one threaded
+instance per thread. Then, it is possible to safely access the user object from each threaded. This
+is not possible in the normal GeneralUserObject unless the access is guarded with a spin mutex.
+
+The `fluid_properties` module was updated to use the `ThreadedGeneralUserObject` to ensure that
+fluid property lookup is always thread safe without the loss of performance.
+
+Note that `ThreadedGeneralUserObject`s are not available when users use pthreads as their threading
+model. The reason is that if a `ThreadedGeneralUserObject` (which runs in a threaded region) created
+a threaded section, there would be `n_threads + n_threads^2` threads active which would yield in a
+performance loss.  If threaded general user objects are needed, they will run properly with OpenMP
+or TBB.
+
+[plotly]: https://plot.ly/
+
+## Working with a different version of libMesh
+
+A new flag has been added to the update_and_rebuild.sh script so that you can work with a different
+version of libmesh. This can be handy when a new feature makes it's way into libMesh but has not
+been rolled out in MOOSE yet. To use this flag, simply checkout the version of libMesh that you want
+and supply "--skip-submodule-update" on the command line to keep that version when you build libMesh.
+
+## GrainTracker Enhancements
+
+For those people working with phase-field and using the GrainTracker, several enhacements and updates
+were merged in during July. First, a significant algorithmic change where merging is performed
```

Comment:
We are using the word "merge" to refer to `git merge` and merging of grains, so I will reword this paragraph.